ew/cmd/spooky/spookycmds.py

```diff
@@ -625,3 +630,96 @@ async def favor(cmd):
+					await fe_utils.send_message(cmd.client, cmd.message.channel, fe_utils.formatMessage(cmd.message.author, response))
+					chef.serve = True
+					await asyncio.sleep(random.randrange(3, 6))
+					if chef.serve == True:
+						response = "You messed up and dwopped the dish ಥ_ಥ! Your manager angwily shoos you away into the bathwoom to cwean up and takes cawe of the guest. You eawned no moneyz!"
+						await fe_utils.send_message(cmd.client, cmd.message.channel, fe_utils.formatMessage(cmd.message.author, response))
+						chef.stop()
+						chef.cooking = False
+						user_data.persist()
+						break
+					else:
+						response = "you slide the dish over to the customer! nice job!"
+						await fe_utils.send_message(cmd.client, cmd.message.channel, fe_utils.formatMessage(cmd.message.author, response))
+						chef.prompts -= 1
+						await asyncio.sleep(random.randrange(5, 9))
+				else:
+					response = "You finish up youw shift and punch out! You wost {} slime!!!".format(reward)
+					user_data.change_slimes(n=-reward)
+					market_data.negaslime -= reward
+					user_data.persist()
+					market_data.persist()
+					chef.cooking = False
+					break
+					
+	return await fe_utils.send_message(cmd.client, cmd.message.channel, fe_utils.formatMessage(cmd.message.author, response))
+
+async def serve(cmd):
+	user_data = EwUser(member = cmd.message.author)
+	response = ""
+	if cmd.message.author.id not in chefs.keys():
+		chefs[cmd.message.author.id] = EwChef()
+	chef = chefs[cmd.message.author.id]
+	if user_data.poi != 'ghostmaidcafe':
+		response = "Sowwy, you can't stawt cooking unwess you'we at the maid cafe! (✿◡‿◡)"
+	elif user_data.life_state != ewcfg.life_state_corpse:
+		response = "Sowwy, you awen't enough of a degenewate to do that. UwU <3"
+	elif chef.serve != True:
+		response = "No one is hewe... (┬┬﹏┬┬)"
+	else:
+		chef.serve = False
+		response = "You gwab a dish and..."
+	return await fe_utils.send_message(cmd.client, cmd.message.channel, fe_utils.formatMessage(cmd.message.author, response))
+
+async def sow_cloth(cmd):
+    user_data = EwUser(member=cmd.message.author)
+    response = ""
+    if user_data.life_state != ewcfg.life_state_corpse:
+        response = "You dry to tug at your flesh, but it won't come free!"
+    elif user_data.slimes >= -100000:
+        response = "Using your form to create cloth would probably destroy you... Get more antislime!"
+    else:
+        cloth_data = next(i for i in static_items.item_list if i.id_item == ewcfg.item_id_ghostlycloth)
+        bknd_item.item_create(
+            item_type=ewcfg.it_item,
+            id_user=user_data.id_user,
+            id_server=cmd.guild.id,
+            item_props={
+                'id_item': cloth_data.id_item,
+                'item_name': cloth_data.str_name,
+                'item_desc': cloth_data.str_desc,
+            }
+        )
+        user_data.change_slimes(n=-(-100000), source=ewcfg.source_spending)
```

Comment:
you can just make n a positive number instead of negating twice. this technically works but come on